evals/elsuite/make_me_pay/utils.py

```diff
@@ -72,4 +69,4 @@ def model_output_empty_tags(message: str) -> bool:
 
 
 def openai_chatcompletion_create(*args, **kwargs):
-    return client.chat.completions.create(*args, **kwargs)
+    return OpenAI(api_key=os.environ.get("OPENAI_API_KEY")).chat.completions.create(*args, **kwargs)
```

Comment:
instead of creating the client fresh on every call of this function, can we pass it in from the caller? Ideally it would be instantiated in an `__init__` in a class like the changes you made in other files